.idea/comp110-worksheet-D.iml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
It's good practice to add IDE-specific files to your `.gitignore` to exclude them from the repo.